sidebarDapp.ts

```diff
@@ -77,6 +78,7 @@ export const sidebarDapp: SidebarsConfig = {
         id: 'dapp/emerald/README',
       },
       items: [
+        'dapp/emerald/chain',
```

Comment:
```suggestion
        'dapp/emerald/network',
```
to keep it consistent.
=> rename chain.mdx to network.mdx